### PR TITLE
Examples updates and some minor typo fixes that broke my parser

### PR DESCRIPTION
**Pull request by jculpon**
Nothing terribly serious here, just wanting to pass some proper attribution
upstream since I forgot to do so when I did a makefile change the other day.

LGTM on my machine, but hopefully that's not enough to :shipit: :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ _False_, so users will not get any surprises when upgrading.
 mention of it in the [README](README.md)
 - Make sure changes do not break backwards compatibility, especially with regards to settings.
 - Only changes that stand to benefit a majority of users or use cases are suitable for contributing back to the main repository. For tweaks that are likely specific to your site or likings, try using `CUSTOM_CSS`.
-    - If doing so would require a CSS selector that isn't supported by the theme, create a patch that adds the necessary CSS selector, not the CSS tweak.
+- If doing so would require a CSS selector that isn't supported by the theme, create a patch that adds the necessary CSS selector, not the CSS tweak.
```
Comment (jculpon) [inline CONTRIBUTING.md]: I think I may have messed this line up, but I don't use github's parser. Happy to revert it if needed.

**Comment by DandyDev** [comment]
Not to be terribly insensitive, but I tried to find your contribution and cannot find it anywhere in the commit history?

**Comment by jculpon** [comment]
@DandyDev No worries, I think the last set that I worked on was fixing packaging issues upstream from this. Specifically, issues involving pelican's build scripts failing on some .deb based linux distros. I can pull up the commits for those when I'm at a work computer.

**Comment by DandyDev** [comment]
Aaaaah, but aren't you confusing the project with the [Pelican project itself](https://github.com/getpelican/pelican)? pelican-bootstrap3 (this repo) has no build scripts :)

**Comment by jculpon** [comment]
I may well be confusing it with pelican proper :)

Now that I'm at a keyboard that won't wake up my roommates, I'll do my best to track down the actual flow of commits. I believe that it has to do with work I did as @dynajamie that I've merged into my personal dotfiles repo at jculpon/dotfiles.

**Comment by jculpon** [comment]
OK, my records show that this was to fix some rendering issues with packaged versions of pelican-bootstrap3 when viewed on @dynamightygames private network. If that's not worth merging, we can close this out and consider the issues fixed by the pelican folks.

**Comment by DandyDev** [comment]
Ok, thanks for looking into it!

**Pull request closed without merging**